### PR TITLE
fixed issue nft-gallery:10004

### PR DESCRIPTION
**Pull request by hamidovz**
As described here: [https://github.com/kodadot/nft-gallery/issues/10004#issuecomment-2041567279](nft-gallery:10004) the issue #10004 on nft-gallery was happening due lack of mimeType property in the final response when request made to `GET /type/endpoint  ` endpoint. i implemented a fix by getting actual image path from ipfs and getting mime-type of the image file. then i assign it to the final response and convert it to the readableStream before sending to the client

### CODE REVIEW

**Comment by exezbcz** [comment]
👀

**Comment by vikiival** [comment]
Hello,
please can you ELI5 your changes?

**Comment by vikiival** [comment]
cc @hamidovz @preschian

**Comment by preschian** [comment]
It's already fixed with these:
- https://github.com/kodadot/nft-gallery/pull/10077
- https://github.com/kodadot/nft-gallery/pull/10087

Will revisit/reopen again if a similar issue occurs
